kernel/ksu.c

```diff
@@ -56,7 +58,7 @@ int __init kernelsu_init(void)
 #ifdef CONFIG_KPROBES
 	ksu_enable_sucompat();
 	ksu_enable_ksud();
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(4, 19, 0)
```

Comment:
Can we remove the condition, and make the `warning` a `info`?

Reply:
Ya bro.. maybe that's better..
I tried on linux 4.9.x without kprobes, and KSU runs normally too

Reply:
4.9 and 4.14 should also work without kprobes, so we shouldn't warn them, too.

Reply:
Ya bro...